src/main/java/ru/itmo/sd/deadliner2bot/model/Chat.java

```diff
@@ -1,7 +1,6 @@
 package ru.itmo.sd.deadliner2bot.model;
 
 import lombok.Getter;
-import lombok.NoArgsConstructor;
```

Comment:
Looks like you can fix commit, where you added this annotation to avoid extra commits

Reply:
What's the profit of amending commits instead of adding new commits? The entire PR changelist only matters.